plugins/samson_ledger/test/lib/samson_ledger/client_test.rb

```diff
@@ -29,10 +40,71 @@
   end
 
   describe ".post_deployment" do
+    before do
+      @asserts = Proc.new {}
```

Comment:
try `let :results` or something else then ... I had a few random segfaults, not sure if that is because of the let or because of singlecov ... but good to know that they are related